README.md

```diff
@@ -13,3 +13,54 @@ The repo contains 1152 message declarations and 32 enum declarations in 192 `.pr
 This includes command, event, rejection, entity, ID, and value object types.
```

Comment:
We would want to add some more proto definitions as our code generation becomes more featureful, or, as we migrate more of it to ProtoData.  

How do we update the above numbers as we update the content of `.proto` files?

Reply:
I did that by manually counting the types. I like the idea of automating it somehow. Maybe, I'll add a counter in ProtoData to highlight this in the build log.

Reply:
But in a separate PR.